ts/express-comparison/encore/README.md

```diff
@@ -0,0 +1,23 @@
+# Encore.ts example
+
+## Developing locally
+
+When you have [installed Encore](https://encore.dev/docs/install), cd into this directory and run:
```

Comment:
Can we add the install commands directly instead of linking, fewer clicks/steps = less friction = more folk actually doing it while they are reading this...